### PR TITLE
Update why-contribute-to-gatsby.md file

### PR DESCRIPTION
**Pull request by vmr1532**
## Description

corrected few of the Grammatical errors on the markdown file like removed useless words like "definitely" from the sentence and changed "Contributing in OSS" to "Contributing to OSS" for the simplicity of the reader.

### CODE REVIEW

File: www/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ If there's a problem with the feedback widgets, please open an issue in the repo
 
 ## Running slow build? (Screenshots placeholder)
 
-If you are not working on starter or site showcase, it might be beneficial to use a placeholder image instead of actual screenshots. It will skip downloading screenshots and generating responsive images for all screenshots and replace them with a placeholder image.
+If you are not working on a starter or site showcase, it might be beneficial to use a placeholder image instead of actual screenshots. It will skip downloading screenshots and generating responsive images for all screenshots and replace them with a placeholder image.
```
Comment (LekoArts) [inline www/README.md]: The current sentence is correct. It refers to the starter showcase.

**Comment by vmr1532** [comment]
I got this mistake by copying the entire sentence to grammerly.

**Comment by LekoArts** [comment]
Hello @vmr1532

Since it’s currently weekend we, the core team, don’t work on the repository and normally your PR would have been reviewed on monday. Since I accidentally saw your PR yesterday I thought: Just request this one change so that the author can edit it until monday and we can directly merge it then.

Unfortunately you've done something absolutely rude that goes against our Code of Conduct (https://www.gatsbyjs.org/contributing/code-of-conduct/): You deleted my review comment. We don’t accept such behavior and future violations will result in a ban from our GitHub organization.

Please hear out other people’s opinions on the internet and be ready to change your opinion/accept change requests upon valid comments.

**Comment by vmr1532** [comment]
I apologize for the mistake that I have done. I ensure you that I won't repeat this mistake again.

**Pull request merged**